docs/ota-client-guide/modules/ROOT/pages/cross-deploy-images.adoc

```diff
@@ -14,18 +14,18 @@ endif::[]
 :icons: font
 :sectnums:
 
-For our recommended production workflow, you will need to move disk images from one account to another from time to time. For example, you might want to send a development build that you're happy with to the QA team, or send that build to the deployment team once it's passed QA. You can do this with our `garage-deploy` tool.
+For our recommended production workflow, you will need to move disk images from one environment to another from time to time. For example, you might want to send a development build that you're happy with to the QA team, or send that build to the deployment team once it's passed QA. You can do this with our `garage-deploy` tool.
 
```

Comment:
I am not sure whether this actually works.

Reply:
Is there a specific reason why you think it would not work or is it because you've never made it work yourself?

Reply:
@lbonn I think he just wanted to make sure it will also work with the new environment feature.